openapi/v2/core-command.yaml

```diff
@@ -71,47 +71,134 @@ components:
         - $ref: '#/components/schemas/BaseResponse'
       description: "A response type for returning a generic error to the caller."
       type: object
-    ConfigResponse:
-      allOf:
-        - $ref: '#/components/schemas/BaseResponse'
-      description: "Provides a response containing the configuration for the targeted service."
+    SettingRequest:
+      description: "Defines new values to be written to device resources, as part of an actuation (put) command to a device"
+      additionalProperties:
+        type: string
+      title: Setting
+      type: object
+      example: { "AHU-TargetTemperature": "28.5", "AHU-TargetBand": "4.0" }
+    BaseReading:
+      description: "A base reading type containing common properties from which more specific reading types inherit. This definition should not be implemented but is used elsewhere to indicate support for a mixed list of simple/binary readings in a single event."
       type: object
       properties:
-        config:
-          description: "A string-ified representation of the service's configuration. For purposes of this specification, a string has been used since configuration structure differs from service to service."
+        apiVersion:
+          description: "A version number shows the API version in DTOs."
           type: string
-    IssueCommandRequest:
+        id:
+          description: "The unique identifier for the reading"
+          type: string
+          format: uuid
+        created:
+          description: "A Unix timestamp indicating when (if) the reading was initially persisted to a database."
+          type: integer
+        origin:
+          description: "A Unix timestamp indicating when the reading was originated at the source device (can support nanoseconds)"
+          type: integer
+        deviceName:
+          description: "The name of the device from which the reading originated"
+          type: string
+        resourceName:
+          description: "The device resource name for the reading"
+          type: string
+        profileName:
+          description: "The device profile name for the reading"
+          type: string
+        valueType:
+          description: "Indicates the datatype of the value property"
+          type: string
+      required:
+        - deviceName
```

Comment:
Please add apiVersion to list of required.

Reply:
Added.